src/inmanta/db/versions/v18.py

```diff
@@ -1,27 +0,0 @@
-"""
```

Comment:
I think we should first get some more clarity about the DB versioning schema before start to remove these files. @wouterdb What do you think?

Reply:
I think we are in agreement on the design once again. Do you think it's fine to remove them now?

Reply:
I think we can now.